### PR TITLE
Updating mnist example to fix minio compatibility

### LINKED ISSUE #100: tensorflow serving not working with minio
👋 Great work guys!!!

Running the mnist example with minio, getting this error:

```
FileSystemStoragePathSource encountered a file-system access error: Could not find base path s3://mybucket/models/myjob-8b52d/export/mnist/ 
```
Any chance you can publish code for `elsonrodriguez/model-server:1.0`?

### PR DESCRIPTION
**Pull request by elsonrodriguez**
Fixes #100, Related to #102

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/examples/108)

### CODE REVIEW

**Comment by ritazh** [comment]
Getting 
```
level=error msg="handle object: can't update services tfworkflow.mnist-myjob-f5202: services \"mnist-myjob-f5202\" is forbidden: User \"system:serviceaccount:tfworkflow:default\" cannot patch services in the namespace \"tfworkflow\""
```

**Comment by ritazh** [comment]
@elsonrodriguez confirm everything works now 🎉

File: mnist/README.md [hunk 1]
```diff
@@ -133,15 +133,18 @@ $ argo list
 NAME   STATUS   AGE   DURATION
 ```
 
-### Creating secrets for our workflow
+### Creating secrets for our workflow and setting AWS variables.
 For fetching and uploading data, our workflow requires S3 credentials. These credentials will be provided as kubernetes secrets:
 
 ```
 export S3_ENDPOINT=s3.us-west-2.amazonaws.com
```
Comment (julienstroheker) [inline mnist/README.md]: Maybe add a comment for minio here, like you did for the next line `#replace with your minio service endpoint ip for default minio installs`

**Comment by julienstroheker** [comment]
@elsonrodriguez @ritazh Works also on my side ! 👍

File: mnist/README.md [hunk 2]
```diff
@@ -133,15 +133,18 @@ $ argo list
...
-export AWS_ENDPOINT_URL=https://${S3_ENDPOINT}
+export AWS_ENDPOINT_URL=https://${S3_ENDPOINT} #use http instead of https for default minio installs
 export AWS_ACCESS_KEY_ID=xxxxx
 export AWS_SECRET_ACCESS_KEY=xxxxx
+export AWS_REGION=us-west-2
```
[the remark this answers is not in the document]
Comment (elsonrodriguez) [reply mnist/README.md]: I've been passing this in to my minio setups and it's been fine. I think it only matters if you explicitly set a region on the minio server.

File: mnist/README.md [hunk 3]
```diff
@@ -133,15 +133,18 @@ $ argo list
 NAME   STATUS   AGE   DURATION
 ```
 
-### Creating secrets for our workflow
+### Creating secrets for our workflow and setting AWS variables.
```
[the remark this answers is not in the document]
Comment (elsonrodriguez) [reply mnist/README.md]: Fixed

File: mnist/README.md (hunk 1 above)
Comment (elsonrodriguez) [reply mnist/README.md]: Added a comment and gave a minio example.

**Pull request merged**